### PR TITLE
(DOCSP-13028) - Replaced Realm.App.Functions with Realm.User.Functions for Realm Web rc1 and onward (including 10.0.0) release

### PR DESCRIPTION
**Pull request by mongomoe**
### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-13028

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/realm/mohammad.hunan/hotfix-realm-web-UserFunctions-instead-of-AppFunctions/web/quickstart.html#call-a-function

### CODE REVIEW

File: source/web/quickstart.txt [hunk 1]
```diff
@@ -182,15 +182,15 @@ defined on the object.
       
       .. code-block:: javascript
          
-         const summed = await app.functions.sum(2,3);
+         const summed = await user.functions.sum(2,3);
```
Comment (cbush) [inline source/web/quickstart.txt:185]: How do you access user? Might be worth mentioning app.currentUser on the line above

File: source/web/quickstart.txt [hunk 2]
```diff
@@ -171,9 +171,9 @@ link users.
 Call a Function
 ---------------
 
-To call a :doc:`function </functions>`, use the ``App.functions`` interface to
-call your serverless functions as if they were a regular JavaScript function
-defined on the object.
+To call a :doc:`function </functions>`, use the ``Realm.User.functions`` interface to
+:ref:`call your serverless functions <web-call-a-function>` as if they were a
+regular JavaScript function defined on the object.
```
Comment (cbush) [inline source/web/quickstart.txt:176]: >they were regular JavaScript functions

**Review by cbush — APPROVED** [review]
Looks good with a couple minor points

File: source/web/quickstart.txt (hunk 1 above)
Comment (mongomoe) [reply source/web/quickstart.txt:185]: The code-block above it actually describes how to access user

File: source/web/quickstart.txt [hunk 3]
```diff
@@ -171,9 +171,9 @@ link users.
 Call a Function
 ---------------
 
-To call a :doc:`function </functions>`, use the ``App.functions`` interface to
-call your serverless functions as if they were a regular JavaScript function
-defined on the object.
+To call a :doc:`function </functions>`, use the ``Realm.User.functions`` interface to
+:ref:`call your serverless functions <web-call-a-function>` as if they were a
```
Comment (mongomoe) [inline source/web/quickstart.txt:175]:
```suggestion
:ref:`call your serverless functions <web-call-a-function>` as if they were 
```

File: source/web/quickstart.txt [hunk 4]
```diff
@@ -171,9 +171,9 @@ link users.
 Call a Function
 ---------------
 
-To call a :doc:`function </functions>`, use the ``App.functions`` interface to
-call your serverless functions as if they were a regular JavaScript function
-defined on the object.
+To call a :doc:`function </functions>`, use the ``Realm.User.functions`` interface to
+:ref:`call your serverless functions <web-call-a-function>` as if they were 
+regular JavaScript function defined on the object.
```
Comment (mongomoe) [inline source/web/quickstart.txt:176]:
```suggestion
regular JavaScript functions defined on the object.
```

File: source/web/quickstart.txt (hunk 2 above)
Comment (mongomoe) [reply source/web/quickstart.txt]: good catch, fixed

**Pull request merged**